src/main/java/org/example/jakarta_labb/entity/Movie.java

```diff
@@ -1,27 +1,28 @@
-package org.example.jakarta_labb;
+package org.example.jakarta_labb.entity;
 
 import jakarta.persistence.Entity;
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
 
+import java.util.UUID;
+
 @Entity
 public class Movie {
 
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private Long id;
-
+    private UUID id;
```

Comment:
I am not sure but I don't think GenerationType.IDENTITY works for UUID.
We are still exposing the primary key. Should we keep the long id and add a new field for UUID?

Reply:
You're right! Maybe i can use "GenerationType.UUID" instead. Or as you mentioned, create a seperate field for UUID.

Reply:
Sounds great to add a new column for UUID :)